conf/evolutions/default/13.sql

```diff
@@ -1,9 +1,17 @@
 # --- !Ups
 
-ALTER TABLE "MenuPerDayPerPerson"
```

Comment:
There's already a 13. This should be `14.sql`

Reply:
It's fixed, and rebased with master